test/distributed/pipelining/test_schedule.py

```diff
@@ -431,6 +432,94 @@ def test_merge_bw(self, test_info):
                 },
                 "stage_to_rank": lambda stage_idx: stage_idx,
                 "num_stages": 2,
+                "simulated_steps": 9,
+            },
+            {
+                "schedule": "v_2_rank_4_stage",
+                "compute": {
+                    0: [
+                        "0F0",
```

Comment:
nit: is it autoformatted to be like this? or is it possible to have the actions together in a row, that would take up less lines.

Reply:
yea, autoformat did this. I dunno if there is a way to override it?